.github/workflows/tag-example-ui.yml

```diff
@@ -4,63 +4,63 @@
 name: Tag example-orchestrator-ui
 
 on:
-  push:
-    tags:
-      - "@orchestrator-ui/orchestrator-ui-components@[0-9]+.[0-9]+.[0-9]+"
-  release:
-    types:
-      - published
-    tags:
-      - "@orchestrator-ui/orchestrator-ui-components@[0-9]+.[0-9]+.[0-9]+"
+    push:
```

Comment:
Changes in this file seem accidental since there are no changes in Prettier / Eslint, i would advise to revert the changes

Reply:
These change are accepted by pipeline, and proposed by prettier. Pipeline has accepted this change, so why should we revert it? Maybe discuss on Thursday?